lib/libzfs_core/libzfs_core.c

```diff
@@ -1111,3 +1112,16 @@ lzc_change_key(const char *fsname, uint64_t crypt_cmd, nvlist_t *props,
 	nvlist_free(ioc_args);
 	return (error);
 }
+
+int
+lzc_reopen(const char *pool_name, boolean_t *scrub_restart)
```

Comment:
Otherwise it has to be dereferenced in ```zpool_reopen_one```. There is no other place that calls ```lzc_reopen``` so I don't see the difference here.

Reply:
The difference is that `lzc_reopen()` is meant to be a more stable interface used by external consumers outside this project. I would move the dereference to `zpool_reopen_one()`

Reply:
Thanks for explanation. I'll change this.